@Test public void addJobManyShouldSucceed() throws Exception {
  j.jenkins.addView(new ListView("aView"));
  FreeStyleProject project1=j.createFreeStyleProject("aProject1");
  FreeStyleProject project2=j.createFreeStyleProject("aProject2");
  assertThat(j.jenkins.getView("aView").getAllItems().size(),equalTo(0));
  assertThat(j.jenkins.getView("aView").contains(project1),equalTo(false));
  assertThat(j.jenkins.getView("aView").contains(project2),equalTo(false));
  final CLICommandInvoker.Result result=command.authorizedTo(Jenkins.READ,View.READ,Job.READ,View.CONFIGURE).invokeWithArgs("aView","aProject1","aProject2");
  assertThat(result,succeededSilently());
  assertThat(j.jenkins.getView("aView").getAllItems().size(),equalTo(2));
  assertThat(j.jenkins.getView("aView").contains(project1),equalTo(true));
  assertThat(j.jenkins.getView("aView").contains(project2),equalTo(true));
}
